frontend/src/index.css

```
@@ -983,4 +983,27 @@ td {
   border-bottom-color: #0ea5a5;
 }
 
+/* ── prefers-reduced-motion ─────────────────────────────── */
+
+@media (prefers-reduced-motion: reduce) {
+  /* Disable skeleton shimmer — show static tint instead */
+  .skeleton {
+    animation: none;
+    background-image: none;
+    background: #e5e7eb;
```

Stylelint (17.12.0)

[error] Overridden property "background-image" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)